README.md

```diff
@@ -3,14 +3,14 @@ This plugin helps you integrate your Laravel WebApp with [Auth0](https://auth0.c
 
 ## Installation
 
-Check our docs page to get a complete guide on how to install it in an existing project or download a pre configured seedproject:
+Check our docs page to get a complete guide on how to install it in an existing project or download a pre-configured seed project:
 
 * Regular webapp: https://auth0.com/docs/quickstart/webapp/laravel/
 * Web API: https://auth0.com/docs/quickstart/backend/php-laravel/
 
 > If you find something wrong in our docs, PR are welcome in our docs repo: https://github.com/auth0/docs
 
-### Setting up a JWKs cache
```

Comment:
This is meant to be JWK (json web keys) and not JWT. JWK is the endpoint that returns the public key to verify the token